static/app/components/questionTooltip.tsx

```diff
@@ -16,7 +16,7 @@ const QuestionIconContainer = styled('span')<ContainerProps>`
 
   & svg {
     transition: 120ms opacity;
-    color: ${p => p.theme.gray300};
+    color: ${p => p.theme.gray200};
```

Comment:
I think this should be `gray300`. `gray200` is too light – it's meant for borders only. We can maybe discuss this further w/ the design team?

Reply:
Yah that’s fine – but let’s not hold up this change for this PR since we should go through and wrap all our Tooltips in wrappers to make the spacing/colors/padding consistent.

Reply:
(I don’t want to make this PR super bloated with every tooltip fix, too)